src/TracyBlueScreenBundle.php

```diff
@@ -4,8 +4,16 @@
 
 namespace Cdn77\TracyBlueScreenBundle;
 
+use Symfony\Bundle\TwigBundle\DependencyInjection\TwigExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 final class TracyBlueScreenBundle extends Bundle
 {
+    public function build(ContainerBuilder $container) : void
+    {
+        parent::build($container);
+
+        $container->registerExtension(new TwigExtension());
```

Comment:
co když appka už tuhle extension registruje? není error registrovat extension dvakrát?

Reply:
ee, to jsem checkoval. Mergne se to v container compilation, takze se registruje jen 1 anyway. -> nenasel jsem teda issue, ktery by s timhle pristupem mohlo nastat

Reply:
ok tak pak mi to dává smysl to sem dát

Reply:
Jako bonus to consumer apps nemusej requirovat (kvuli missing requirements checkum), protoze v kodu twigbundle mit nebudou